assets/src/domain/blocks/event-attendees/controls/ArchiveSetttings.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { __ } from '@wordpress/i18n';
+import { ToggleControl } from '@wordpress/components';
+
+import { AttendeesEditProps } from '../types';
+
+const ArchiveSetttings: React.FC<AttendeesEditProps> = ({ attributes: { displayOnArchives }, setAttributes }) => {
+	return (
+		<ToggleControl
+			label={__('Display on Archives', 'event_espresso')}
+			checked={displayOnArchives}
+			onChange={(displayOnArchives): void => setAttributes({ displayOnArchives })}
+			help={
+				displayOnArchives
+					? __('Attendees are shown whenever this post is listed in an archive view.', 'event_espresso')
+					: __('Attendees are hidden whenever this post is listed in an archive view.', 'event_espresso')
```

Comment:
not strictly related to this PR, just thoughts out loud:
I've seen this pattern in a few places, it would handy to have a utility function such as the result would be smth like this:
```
const help = toggleText('Attendees are %s whenever this post is listed in an archive view.', ['shown','hidden'], displayOnArchives )
```

Reply:
As mentioned earlier, `i18n` functions do not work that way 😄